server/service/src/service_provider.rs

```diff
@@ -150,12 +156,15 @@ pub struct ServiceContext {
     pub store_id: String,
 }
 
+
+
 impl ServiceProvider {
     // TODO we should really use `new` with processors_trigger, we constructs ServiceProvider manually in tests though
     // and it would be a bit of refactor, ideally setup_all and setup_all_with_data will return an instance of ServiceProvider
     // {make an issue}
     pub fn new(connection_manager: StorageConnectionManager, app_data_folder: &str) -> Self {
-        ServiceProvider::new_with_triggers(
+
+        return ServiceProvider::new_with_triggers(
```

Comment:
Is this 'return' added by some prettier/clipy config ? I actually prefer no return statement now (unless early return), it removes extra bit of verboseness.